### PR TITLE
Add information about console commands for connect/disconnect to networking settings

### PR DESCRIPTION
**Pull request by lmbr-pip**
…ork settings page

Signed-off-by: Pip Potter <61438964+lmbr-pip@users.noreply.github.com>

## Change summary
Adds missing information about console commands for networking. Should link to https://github.com/o3de/o3de.org/pull/1916 but that change is currently just for development. 

Attempts to provide some basic context.

Tested with ```hugo serve --watch=false```

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/networking/settings.md [hunk 1]
```diff
@@ -5,9 +5,41 @@ linktitle: Network Settings
 ---
 
 ## Overview
-This page documents [console variables](/docs/user-guide/appendix/cvars) and settings for AzNetworking and Multiplayer behavior.
+## Overview
+This page documents [console variables](/docs/user-guide/appendix/cvars) and other settings that control Networking and Multiplayer behavior.
+
+## Networking commands
+These are special console commands that control the flow of networked and multiplayer games.
```
Comment (lmbr-pip) [inline content/docs/user-guide/networking/settings.md:12]:
```suggestion
These are special console commands that are used to control the flow of networked and multiplayer games. 
```

File: content/docs/user-guide/networking/settings.md [hunk 2]
```diff
@@ -34,6 +66,8 @@ This page documents [console variables](/docs/user-guide/appendix/cvars) and set
 
 
 ## Server settings
+The following CVars control server behaviour.
```
Comment (AMZN-Gene) [inline content/docs/user-guide/networking/settings.md:69]: behavior (without u)

File: content/docs/user-guide/networking/settings.md [hunk 3]
```diff
@@ -5,9 +5,41 @@ linktitle: Network Settings
...
+And the client's configuration file should contain:
+
+```
+connect
+```
+
+Commands in configuration files can be passed to launchers using the `console-command-file` option, for example `MultiplayerSample.ServerLauncher.exe --console-command-file=launch_server.cfg`.
 
 ## Client settings
+The following CVars control client behaviour.
```
Comment (AMZN-Gene) [inline content/docs/user-guide/networking/settings.md:41]: behavior (without u)

**Review by AMZN-Gene — APPROVED** [review]

File: content/docs/user-guide/networking/settings.md [hunk 4]
```diff
@@ -5,9 +5,41 @@ linktitle: Network Settings
...
+
+## Networking commands
+These are special console commands that control the flow of networked and multiplayer games. 
+
+| Setting                | Description                                                                                | Default | Notes |
+|------------------------|--------------------------------------------------------------------------------------------|---------|-------|
+| host                   | Opens a multiplayer connection as a host for other clients to connect to.                  |||                                                                      ||
+| connect                | Opens a multiplayer connection to a host                                                   |||
+| disconnect             | Disconnects any open multiplayer connections                                               ||| 
+| LoadLevel              | Unloads the current level and loads a new one with the given asset name.                   |||
```
Comment (AMZN-Gene) [inline content/docs/user-guide/networking/settings.md:19]: Ah, nice call adding loadlevel. It's not multiplayer specific, but basically essential to anyone working with multiplayer (or looking at this page)

**Review by AMZN-Gene — APPROVED** [review]

File: content/docs/user-guide/networking/settings.md (hunk 3 above)
Comment (kberg-amzn) [reply content/docs/user-guide/networking/settings.md:41]: Behaviour has a u in it for.. I think everyone who speaks English except folks who live in the US?

File: content/docs/user-guide/networking/settings.md [hunk 5]
```diff
@@ -5,9 +5,41 @@ linktitle: Network Settings
...
+## Overview
+This page documents [console variables](/docs/user-guide/appendix/cvars) and other settings that control Networking and Multiplayer behavior.
+
+## Networking commands
+These are special console commands that control the flow of networked and multiplayer games. 
+
+| Setting                | Description                                                                                | Default | Notes |
+|------------------------|--------------------------------------------------------------------------------------------|---------|-------|
+| host                   | Opens a multiplayer connection as a host for other clients to connect to.                  |||                                                                      ||
+| connect                | Opens a multiplayer connection to a host                                                   |||
```
Comment (kberg-amzn) [inline content/docs/user-guide/networking/settings.md:17]: This command also optionally takes a hostname/IP address and port number to connect to if you don't want to just connect to localhost on the default port.

File: content/docs/user-guide/networking/settings.md (hunk 2 above)
Comment (lmbr-pip) [reply content/docs/user-guide/networking/settings.md:69]: Opps, have UK spelling on by default. Will fix.

File: content/docs/user-guide/networking/settings.md (hunk 5 above)
Comment (lmbr-pip) [reply content/docs/user-guide/networking/settings.md]: Added

File: content/docs/user-guide/networking/settings.md [hunk 6]
```diff
@@ -5,9 +5,41 @@ linktitle: Network Settings
...
-This page documents [console variables](/docs/user-guide/appendix/cvars) and settings for AzNetworking and Multiplayer behavior.
+## Overview
+This page documents [console variables](/docs/user-guide/appendix/cvars) and other settings that can control Networking and Multiplayer behavior.
+
+## Networking commands
+These are console commands that control the flow of networked and multiplayer games, or will be required to for a networked game or a simulation.
+
+| Setting                | Description                                                                                | Notes                                                                                                                                      |
+|------------------------|--------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| host                   | Opens a multiplayer connection as a host for other clients to connect to.                  ||                                                                      ||
```
Comment (allisaurus) [inline content/docs/user-guide/networking/settings.md:16]: maybe as a note for this and `disconnect` we can add a note like "Takes no arguments".

**Review by allisaurus — COMMENTED** [review]
thank you for updating this! one question about command args

**Review by kberg-amzn — APPROVED** [review]

File: content/docs/user-guide/networking/settings.md (hunk 6 above)
Comment (lmbr-pip) [reply content/docs/user-guide/networking/settings.md]: Added params and a some notes.

File: content/docs/user-guide/networking/settings.md [hunk 7]
```diff
@@ -38,7 +37,7 @@ connect
 Commands in configuration files can be passed to launchers using the `console-command-file` option, for example `MultiplayerSample.ServerLauncher.exe --console-command-file=launch_server.cfg`.
 
 ## Client settings
-The following CVars control client behavior.
+The following [console variables](http://localhost:1313/docs/user-guide/appendix/cvars/) control client behavior.
```
Comment (allisaurus) [inline content/docs/user-guide/networking/settings.md:40]: broken link - need to remove "http://localhost:1313"

File: content/docs/user-guide/networking/settings.md [hunk 8]
```diff
@@ -66,7 +65,7 @@ The following CVars control client behavior.
 
 
 ## Server settings
-The following CVars control server behavior.
+The following [console variables](http://localhost:1313/docs/user-guide/appendix/cvars/) control server behavior.
```
Comment (allisaurus) [inline content/docs/user-guide/networking/settings.md:68]: broken link - need to remove "http://localhost:1313"

File: content/docs/user-guide/networking/settings.md (hunk 7 above)
Comment (lmbr-pip) [reply content/docs/user-guide/networking/settings.md:40]: Sigh, copy-n-pasta fail.

File: content/docs/user-guide/networking/settings.md (hunk 8 above)
Comment (lmbr-pip) [reply content/docs/user-guide/networking/settings.md:68]: Sigh, copy-n-pasta fail.

File: content/docs/user-guide/networking/settings.md [hunk 9]
```diff
@@ -5,9 +5,40 @@ linktitle: Network Settings
...
+And the client's configuration file should contain:
+
+```
+connect
+```
+
+Commands in configuration files can be passed to launchers using the `console-command-file` option, for example `MultiplayerSample.ServerLauncher.exe --console-command-file=launch_server.cfg`.
 
 ## Client settings
+The following [console variables](http://localhost:1313/docs/user-guide/appendix/cvars/) control client behavior.
```
Comment (lmbr-pip) [inline content/docs/user-guide/networking/settings.md:40]:
```suggestion
The following [console variables](/docs/user-guide/appendix/cvars/) control client behavior.
```

File: content/docs/user-guide/networking/settings.md [hunk 10]
```diff
@@ -34,6 +65,8 @@ This page documents [console variables](/docs/user-guide/appendix/cvars) and set
 
 
 ## Server settings
+The following [console variables](http://localhost:1313/docs/user-guide/appendix/cvars/) control server behavior.
```
Comment (lmbr-pip) [inline content/docs/user-guide/networking/settings.md:68]:
```suggestion
The following [console variables](/docs/user-guide/appendix/cvars/) control server behavior.
```

**Review by allisaurus — APPROVED** [review]

File: content/docs/user-guide/networking/settings.md [hunk 11]
```diff
@@ -5,9 +5,40 @@ linktitle: Network Settings
 ---
 
 ## Overview
-This page documents [console variables](/docs/user-guide/appendix/cvars) and settings for AzNetworking and Multiplayer behavior.
+This page documents [console variables](/docs/user-guide/appendix/cvars/) and other settings that can control Networking and Multiplayer behavior.
+
+## Networking commands
+The following [console functor](/docs/user-guide/programming/az-console/#console-functors-cfuncs) commands control the flow of networked and multiplayer games, or will be required to for a networked game or a simulation.
```
Comment (willihay) [inline content/docs/user-guide/networking/settings.md:11]: I'm not sure what information that the phrase "or will be required to for a networked game or a simulation" is adding to the first part of the sentence. Can you explain?

File: content/docs/user-guide/networking/settings.md [hunk 12]
```diff
@@ -5,9 +5,40 @@ linktitle: Network Settings
...
+
+| Setting                | Description                                                               | Parameters                                                                      | Notes                                                                                    |
+|------------------------|---------------------------------------------------------------------------|---------------------------------------------------------------------------------|------------------------------------------------------------------------------------------|
+| host                   | Opens a multiplayer connection as a host for other clients to connect to. |                                                                                 |                                                                                          |
+| connect                | Opens a multiplayer connection to a host                                  | *(Optional)* IP address and port, separated by ':'. For example, `0.0.0.0:1234` | Defaults to `cl_serveraddr`:`cl_serverport`                                              |
```
Comment (willihay) [inline content/docs/user-guide/networking/settings.md:16]:
```suggestion
| connect                | Opens a multiplayer connection to a host.                                 | *(Optional)* IP address and port, separated by ':'. For example, `0.0.0.0:1234`. | Defaults to `cl_serveraddr`:`cl_serverport`.                                             |
```

File: content/docs/user-guide/networking/settings.md [hunk 13]
```diff
@@ -5,9 +5,40 @@ linktitle: Network Settings
...
+| disconnect             | Disconnects any open multiplayer connections                              |                                                                                 |                                                                                          |
```
Comment (willihay) [inline content/docs/user-guide/networking/settings.md:17]:
```suggestion
| disconnect             | Disconnects any open multiplayer connections.                             |                                                                                 |                                                                                          |
```

File: content/docs/user-guide/networking/settings.md [hunk 14]
```diff
@@ -5,9 +5,40 @@ linktitle: Network Settings
...
+| LoadLevel              | Unloads the current level and loads a new one with the given asset name.  | *(Required)* Path to a level file                                               | Command is not specific to network or multiplayer but used for all games and simulations |                                                                                         |
```
Comment (willihay) [inline content/docs/user-guide/networking/settings.md:18]:
```suggestion
| LoadLevel              | Unloads the current level and loads a new one with the given asset name.  | *(Required)* Path to a level file.                                              | Command is not specific to network or multiplayer but used for all games and simulations.                                                                                         | 
```

File: content/docs/user-guide/networking/settings.md [hunk 15]
```diff
@@ -5,9 +5,40 @@ linktitle: Network Settings
...
+| disconnect             | Disconnects any open multiplayer connections                              |                                                                                 |                                                                                          |
+| LoadLevel              | Unloads the current level and loads a new one with the given asset name.  | *(Required)* Path to a level file                                               | Command is not specific to network or multiplayer but used for all games and simulations |                                                                                         | 
+| sv_launch_local_client | Launches a local client and connects to this host server.                 |                                                                                 |                                                                                          | only works if currently hosting |
```
Comment (willihay) [inline content/docs/user-guide/networking/settings.md:19]:
```suggestion
| sv_launch_local_client | Launches a local client and connects to this host server.                 |                                                                                 | Only works if currently hosting. |
```

File: content/docs/user-guide/networking/settings.md [hunk 16]
```diff
@@ -5,9 +5,40 @@ linktitle: Network Settings
...
+
+These console commands can be executed dynamically via the [console command line](/docs/user-guide/editor/console/) or placed within a console command configuration file, usually with the `.cfg` suffix. Commands will be
+executed in order written.
```
Comment (willihay) [inline content/docs/user-guide/networking/settings.md:22]:
```suggestion
These console commands can be executed dynamically via the [console command line](/docs/user-guide/editor/console/) or placed within a console command configuration file, usually with the `.cfg` suffix. Commands will be executed in the order written.
```

**Review by willihay — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/networking/settings.md (hunk 11 above)
Comment (willihay) [reply content/docs/user-guide/networking/settings.md:11]: Also, "required to for" is an awkward phrase.

Comment (lmbr-pip) [reply content/docs/user-guide/networking/settings.md:11]: Should be something "or will be required to complete setup'. Trying to highlight that LoadLevel is required here for setup but is not specific for Networked game or simulation. Will try and re-phrase, but happy to take a suggestion.

Comment (lmbr-pip) [reply content/docs/user-guide/networking/settings.md]: I simplified this text.

**Review by willihay — APPROVED** [review]

**Comment by willihay** [comment]
Nice update.

**Pull request merged**